docusaurus.config.js

```diff
@@ -213,7 +213,7 @@ const config = {
           },
           {
             from: "/Explorer-Backend-Starter-Guide.html",
-            to: "/full-node/explorer-backend",
+            to: "/infrastructure/explorer-backend",
```

Comment:
@nop33 it's here :) right next to you

Reply:
![image](https://github.com/user-attachments/assets/7a79eae9-6607-45b6-8266-13895c87e8e0)

Reply:
This is not what I mean. Here you changed an existing redirect to a new one.

What I mean is that we need both:
`/Explorer-Backend-Starter-Guide.html` => `/infrastructure/explorer-backend`
`/full-node/explorer-backend` => `/infrastructure/explorer-backend`

otherwise the original redirect breaks.

Reply:
```ts
{
            from: "/Explorer-Backend-Starter-Guide.html",
            to: "/infrastructure/explorer-backend",
},
{
            from: "/full-node/explorer-backend",
            to: "/infrastructure/explorer-backend",
}
```

Reply:
ooooooooh ok, my bad

Reply:
@nop33 fix pushed